tests/integrations/openai/test_openai.py

```diff
@@ -0,0 +1,187 @@
+import pytest
+from openai import OpenAI, Stream, OpenAIError
+from openai.types import CompletionUsage, CreateEmbeddingResponse, Embedding
+from openai.types.chat import ChatCompletion, ChatCompletionMessage, ChatCompletionChunk
+from openai.types.chat.chat_completion import Choice
+from openai.types.chat.chat_completion_chunk import ChoiceDelta, Choice as DeltaChoice
+from openai.types.create_embedding_response import Usage as EmbeddingTokenUsage
+
+from sentry_sdk import start_transaction
+from sentry_sdk.integrations.openai import (
+    OpenAIIntegration,
+    COMPLETION_TOKENS_USED,
+    PROMPT_TOKENS_USED,
+    TOTAL_TOKENS_USED,
+)
+
+from unittest import mock
+
```

Comment:
Please add some tests that check for PII. Something like:
* init() with OpenAIIntegration() -> no prompts
* init(send_default_pii=True) with OpenAIIntegration() -> prompts
* init(send_default_pii=True) with OpenAIIntegration(include_prompts=False) -> no prompts
* init() or init(send_default_pii=False) with OpenAIIntegration(include_prompts=True) -> no prompts

(we will add a note in the integrations docs to make sure users get this)